hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/container/common/helpers/DeletedBlocksTransactionInfoWrapper.java

```diff
@@ -39,6 +39,13 @@ public DeletedBlocksTransactionInfoWrapper(long txID, long containerID,
     this.count = count;
   }
 
+  DeletedBlocksTransactionInfoWrapper() {
```

Comment:
Remove this unused method.

Reply:
Thanks a lot for the comment @szetszwo The reason why I included the default constructor is because in case if its not there then `TestDeletedBlocksTxnShell` will thrown the following error :- 
```
java.io.IOException: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `org.apache.hadoop.hdds.scm.container.common.helpers.DeletedBlocksTransactionInfoWrapper` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 2, column: 3] (through reference chain: java.lang.Object[][0])
```

Which aims to to indicate that Jackson, is unable to deserialize an instance of `DeletedBlocksTransactionInfoWrapper` because it lacks a **default constructor**. Jackson requires a default (no-argument) constructor to create instances of a class during deserialization.

To resolve this issue, we have added a default constructor to the `DeletedBlocksTransactionInfoWrapper`

Reply:
The no-args constructor can be avoided by using Jackson annotations:

```
  @JsonCreator
  public DeletedBlocksTransactionInfoWrapper(
      @JsonProperty("txID") long txID,
      @JsonProperty("containerID") long containerID,
      @JsonProperty("localIdList") List<Long> localIdList,
      @JsonProperty("count") int count
```

The one with annotations might be faster at runtime, as properties can be set without reflection.  It also allows JSON property names to be kept constant even if member variable is renamed for some reason.

Reply:
Thanks for the idea @adoroszlai it worked!
I have made the changes!